src/app/shared/validationMessages.model.ts

```diff
@@ -133,19 +133,19 @@ export const validationMessages = {
     required: 'This field cannot be empty.',
     minlength: 'Image Path is too short (minimum 3 characters).',
     maxlength: 'Image Path is too long (max 128 characters).',
-    pattern: `The namespace and name of the image repository, separated by a '/'. The name of the image repository cannot contain additional '/'.`,
+    pattern: `Must have the form <namespace>/<name>. The <name> cannot contain additional '/'.`,
   },
   repoNameWithSlashesImagePath: {
     required: 'This field cannot be empty.',
-    minlength: 'Image Path is too short (minimum 3 characters). ',
+    minlength: 'Image path is too short (minimum 3 characters).',
     maxlength: 'Image Path is too long (max 128 characters).',
-    pattern: `The namespace and name of the image repository, separated by a '/'. The name of the image repository may contain additional '/'.`,
+    pattern: `Must have the form <namespace>/<name>. The <name> may contain additional '/'.`,
   },
   privateAmazonImagePath: {
     required: 'This field cannot be empty.',
     minlength: 'Image Path is too short (minimum 3 characters).',
     maxlength: 'Image Path is too long (max 128 characters).',
-    pattern: `The namespace and name of the image repository, separated by a '/'. The namespace must be empty (use '_'). The name of the image repository may contain additional '/'.`,
+    pattern: `Must have the form <namespace>/<name>. The namespace must be empty (use '_').`,
```

Comment:
I find this a little confusing: it must be of the form `<namespace>/<name>`, but leave the namespace empty, but don't leave it empty, use `_`.

How about `Must have the form _/<name>`, maybe with an explanation.